src/controllers/check.your.answers.controller.ts

```diff
@@ -66,8 +66,7 @@ export const post = async (req: Request, res: Response, next: NextFunction) => {
       ? appData[OverseasEntityKey] as string
       : await createOverseasEntity(req, session, transactionID);
 
-    // TODO: Missing last put call to submit OE, it will be done on ROE-1441.
-    // Note: this will be removed in the future when all PUT calls have been set correctly on the others pages.
+    await updateOverseasEntity(req, session);
```

Comment:
I think we only need to do this PUT call if the 'save and resume' feature is active.

Looking at the above logic might read better if we just have one 'if' check on the feature flag and two execution blocks - one for when we're in save and resume mode and the other when not.

Reply:
Good spot, but I ill not use a single if condition. It is always better using the `cost` instead of `let`

Reply:
👍 I suppose what I'm saying is that using the ternary operator here means multiple checks - an if ... else block would make flow easier to read I think, but will wait to see the next updates.

Reply:
Done, it will be refactored anyway soon